GitHubSearch/Services/Realm/RealmService.swift

```diff
@@ -17,7 +17,7 @@ struct RealmService {
         Realm.Configuration.defaultConfiguration = realmConfig
     }
 
-    private var realm: Realm {
+    internal var realm: Realm {
```

Comment:
`internal` is unnecessary here, just removing `private` will do the job 🙂

Reply:
Can you look at it here and in other places?

Reply:
Default access level is internal, so internal is unnecessary here.
See you checking the code with SwiftLint. I now also start to use it in out projects. Check out too many problems.
Excuse me for my poor english. I need to use Google translation for communication